src/analytics/is-analytics-supported-factory.ts

```diff
@@ -0,0 +1,19 @@
+import { ɵisSupportedError } from '@angular/fire';
+import { isSupported } from '@firebase/analytics';
```

Comment:
@robertIsaac The firebase package is set up only for use from the main package and not from the scoped packages. It can work at times but in general it is not recommended and can result in errors. 

```suggestion
import { isSupported } from 'firebase/analytics';
```